src/main/java/seedu/duke/commands/DeleteModuleCommand.java

```diff
@@ -0,0 +1,58 @@
+package seedu.duke.commands;
+
+import seedu.duke.common.Messages;
+import seedu.duke.exceptions.CommandException;
+import seedu.duke.module.ModuleList;
+import seedu.duke.ui.UI;
+
+import java.util.ArrayList;
+
+import static java.util.Objects.requireNonNull;
+
+public class DeleteModuleCommand extends Command {
+
+    public static final String MESSAGE_PROMPT1 = "Which modules would you like to delete?\n";
+    public static final String MESSAGE_PROMPT2 = "Please enter the indices of the modules you would like to delete.\n"
+            + "Separate indices with a blank space.";

```

Comment:
Maybe these constants can be combined and their name can be more descriptive? e.g. MESSAGE_DELETE_PROMPT